packages/common/src/index.ts

```diff
@@ -860,7 +860,7 @@ export default class Common extends EventEmitter {
     // Logic: if accumulator is still null and on the first occurence of
     // a block greater than the current hfBlock set the accumulator,
     // pass on the accumulator as the final result from this time on
-    const nextHfBlock = this.hardforks().reduce((acc: BN, hf: any) => {
+    const nextHfBlock = this.hardforks().reduce((acc: BN | null, hf: any) => {
```

Comment:
What is the effect of this change?

Reply:
This is being more accurate with regards to the accumulator type and doesn't change the actual logic (in line 865 we were already testing for `acc === null`, which implies that the ). This became necessary when the `this.hardforks()` return type changed from `any[]` to `HardforkParams[]`